wagtail/admin/templatetags/wagtailadmin_tags.py

```diff
@@ -82,14 +81,32 @@ def breadcrumbs(
         .specific(),
         "current_page": page,
         "is_expanded": is_expanded,
-        "page_perms": page_perms,
         "querystring_value": querystring_value or "",
         "trailing_breadcrumb_title": trailing_breadcrumb_title,  # Only used in collapsible breadcrumb templates
         "url_name": url_name,
         "url_root_name": url_root_name,
     }
 
 
+@register.inclusion_tag("wagtailadmin/shared/breadcrumbs.html")
+def breadcrumbs_admin(
+    label,
+    url,
```

Comment:
I think this needs to accept a list of label and URLs in order to be truly reusable, but that means the list needs to be defined in the view first. Which probably isn't a bad idea as it helps you prevent writing too much logic in the templates.

Reply:
Not sure if I am missing something but you cannot easily pass in a list

```
{% breadcrumbs_simple items=[view.verbose_name_plural|capfirst, view.index_url] %}
```

This will not work as Django templates cannot parse this, instead it would have to be passed in as a variable which makes it harder to re-use.

Maybe we could handle both cases, where you can pass in a url & label OR items.

```
{% breadcrumbs_simple items=my_items %}
```

But again - trying to solve for a simple case for now and then we can build on it later.

Reply:
Yes, maybe I worded it poorly, but it's exactly what I mean (having to define the list in the view first) 😄 

Accepting multiple parameters don't seem very clean to me (i.e. what should happen if you define both?). That said, I don't see any other way if we want to support both use cases (defining the item directly in the template, or multiple items from the view).

> But again - trying to solve for a simple case for now and then we can build on it later.

Yep, it's just that I can't think of a case where having breadcrumbs is that useful if there's only two items in it. In most cases, you want to add more breadcrumb items as you go deeper. At that point, I think it's better to have one (and only one) way to define the items, that is by defining the list in the view. That means when you need to find where the breadcrumb items are defined, you only have to look in one place.

Reply:
I suppose the idiomatic Django way to pass a list of links purely in template code would be an opening/closing tag pair with inner tags:

    {% breadcrumbs %}
        {% breadcrumb_item "Settings" view.index_url %}
        {% breadcrumb_item "Sites" view.sites_url %}
    {% endbreadcrumbs %}

Reply:
Ok. Can try to go down that route. Just might take longer to solve - not sure I can take the time to do a more full solution. 

My understanding was that we only needed (for now) basic breadcrumbs that solved a very basic need. 

Two items
- home (aka dashboard)
- custom one (e.g. the snippet listing or the ModelAdmin listing)

With a buffer of a third with the trailing breadcrumb title slot (which can be any label). 

I was trying to solve for that niche (as described on the issue raised) instead of solve for breadcrumbs that can do anything. Hence avoiding a larger re-write of the breadcrumbs template.